manifests/extension/tidy.pp

```diff
@@ -0,0 +1,60 @@
+# == Class: php::extension::tidy
+#
+# Install the tidy PHP extension
+#
+# === Parameters
+#
+# [*ensure*]
+#   The version of the package to install
+#   Could be "latest", "installed" or a pinned version
+#   This matches "ensure" from Package
+#
+# [*package*]
+#   The package name in your provider
+#
+# [*provider*]
+#   The provider used to install the package
+#
+# [*inifile*]
+#   The path to the extension ini file
+#
+# [*settings*]
+#   Hash with 'set' nested hash of key => value
+#   set changes to agues when applied to *inifile*
+#
+# === Variables
+#
+# No variables
+#
+# === Examples
+#
+#  include php::extension::tidy
+#
+# === Authors
+#
+# Christian "Jippi" Winther <jippignu@gmail.com>
```

Comment:
All I did is copy/paste and find/replace. :) 

At best I could add my name below Christian's but even that is doubtful.